src/main/java/ladder/view/ResultView.java

```diff
@@ -44,10 +49,37 @@ private String printUsedOrNot(Point point) {
   }
 
 
+  private String printPrize(Prizes prizes) {
+    return prizes.prizes()
+        .map(prize -> prize.prize())
```

Comment:
```suggestion
        .map(Prize::prize)
```
람다로 좀 더 간결하게 표현 할 수도 있습니다. 😀